host.go

```diff
@@ -360,10 +361,17 @@ func (h *Host) Create(name string) error {
 		return err
 	}
 
+	// create the instance
 	if err := h.Driver.Create(); err != nil {
 		return err
 	}
 
+	// install docker
+	if err := h.Provision(); err != nil {
```

Comment:
It would be great to have a flag to disable this step. On the OpenStack driver we had a flag `--openstack-docker-install` useful when we use an image with docker already installed and well configured.

`--no-provision` maybe ?